src/platform/plugins/shared/discover/public/application/main/state_management/tabs_storage_manager.ts

```diff
@@ -400,6 +400,7 @@ export const createTabsStorageManager = ({
     const persistedTabs = persistedDiscoverSession?.tabs.map((tab) =>
       fromSavedObjectTabToTabState({ tab })
     );
+
```

Comment:
```suggestion
```
Only suggesting to remove it because I know it will create conflicts with another incoming PR that refactors this 🙂